translate/rus.json

```diff
@@ -48,7 +48,7 @@
 "Benchmark over.":"Бенчмарк завершен.",
 "benchmark":"бенчмарк",
 "black mates":"мат черными",
-"black wins":"чёрные победы",
+"black wins":"победы чёрных",
```

Comment:
Lines 51 and 436 just veeeeery slowly got on my nerves and I changed them hehe:)